src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -165,20 +166,28 @@ private static String extractVersionNumber(String inputString) {
 
     /**
      * Verifies that the test properties specified in test.properties file allows for HTML
-     * regeneration via God mode to work smoothly (i.e all test HTML files are correctly regenerated,
+     * regeneration via GodMode to work smoothly (i.e all test HTML files are correctly regenerated,
      * strings that need to be replaced with placeholders are correctly replaced, and
      * strings that are not supposed to be replaced with placeholders are not replaced).
      */
     public static void verifyReadyForGodMode() {
+        if (IS_GODMODE_ENABLED && isCiEnvironment() && !isGodModeTest()) {
+            fail("GodMode should only be run locally, not in a CI environment. Please revert the change "
+                    + "to the test properties template file that enabled GodMode in CI.");
```

Comment:
Actually, since we're talking about CI, doing it in just one `@BeforeSuite` should be sufficient.